en-devonfw.dict

```diff
@@ -1206,3 +1201,30 @@ jre
 propertymerge
 autocompletion
 Chocolatey
+
+FreeMarker
+JSON
+JUnit 
+Ionic
+IONIC
+JAXB 
+XML
+POM
+OpenApi
+JSON
+TS
+TypeScript
+POC
+IDE
+CMD
+SQL
+REST
+TCP
+URL
+csrf
+CSRF
+CLI
+SOAP
+WSDL
+EXT JS
```

Comment:
What is this?

Reply:
Technical words which should not be detected as code